doc/performance-investigations/0004-update-to-python-38/summary.md

```diff
@@ -0,0 +1,47 @@
+# Update to Python 3.8
+
+Following gevent now supporting Python 3.8 (as of gevent 1.5) we are now able to upgrade Python.
+There will be performance implications with both Pythion 3.8 and dependent the gunicorn/gevent libraries.
+
+https://github.com/ONSdigital/eq-questionnaire-runner/tree/update-to-python-38
+
+## Benchmark version
+
+| Tag | Identifying Commit |
+|--------|-------|
+| latest | 7c865648ad1665583ee0a6a1738e5a4a70e28f80 |
+
+## Benchmark profile
+
+| Option | Value |
+|--------|-------|
+| Requests file | census_household_gb_eng.json |
+| Run time | 60 mins |
+| Instances | 1 |
+| User wait time minimum | 1 |
+| User wait time maximum | 2 |
+| Clients | 100 |
+| Hatch rate | 100 |
+
+## Runner images
+| Type      | Tag    | Commit | Date |
+|-----------|--------|--------|------|
+| Baseline  | latest  | 4bd042ae1e1ab75f4f4f377c0a26a9124ba30207 | Tue May 5 12:51:10 2020 +0100 |
+| Investigation Candidate | update-to-python-38 | 29ddee54edfc60d0ee7ecc04944911e21bb6c4dd | Tue May 5 09:41:30 2020 +0100 |
+
+## Results
+
+Results based on 99th percentile timings.
+
+| Metric | Baseline | Investigation |
+|--------|----------|--------------|
+| Questionnaire GETs | 587 | 393 |
+| Questionnaire POSTs | 730 | 431 |
+| All requests | 690 | 380 |
+
+## Supporting Observations
+![](0004-response-rates-and-latencies.png)
```

Comment:
This renders to nothing for me

Reply:
Refresh your browser. Is there another way to insert the image?

Reply:
This is the url generated in the diff on Github https://github.com/ONSdigital/eq-survey-runner-benchmark/blob/master/doc/performance-investigations/0004-update-to-python-38/0004-response-rates-and-latencies.png - it's pointing to master not this branch, so I suspect it's a GitHub issue.